requirements.txt

```diff
@@ -1,4 +1,4 @@
 beautifulsoup4==4.9.1
 cryptography==39.0.2
```

Comment:
I was looking at the previous commits after the last release, and I actually see that the cryptography package was added to the requirements.txt and then downgraded - is there a reason why you added it? It is my fault for missing it as I approved it. (https://github.com/splunk-soar-connectors/ssh/pull/15/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R2) Just wondering if it is possible to remove this package from requirements.txt or not?

Reply:
UPDATE: Bumped cryptograpgy to 41.0.6 due to new vulnerability detected in previous versions.